### ISSUE TITLE
Request for training code & native-resolution question

### ISSUE THREAD

**Issue by JihwanEom** [deepglint/MVT#3]
Hello,

First of all, thank you for the excellent work and for sharing this repository. I read the RICE-ViT paper(https://arxiv.org/abs/2507.20025) with great interest—it was truly impressive.

If you don’t mind, I had two quick questions:

---

### 1) Training code release plan
- Do you have plans to **release the training code for RICE-ViT**?
### 2) Native-resolution training for RICE-ViT?
- In the LLaVA-OneVision-1.5 paper ([arXiv:2509.23661](https://www.arxiv.org/abs/2509.23661)), the authors mention training with **native resolution**. I’m curious whether RICE-ViT is also trained with native resolution regime starting from pre-training stage (cluster classification loss) for LLaVA-OV-1.5.
---

Apologies for the interruption, and thank you so much for your time. 
Even a brief reply would be incredibly helpful.

Best,
Jihwan
